packages/jest-util/src/globsToMatcher.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import micromatch = require('micromatch');
+import type {Config} from '@jest/types';
+import replacePathSepForGlob from './replacePathSepForGlob';
+
+const globsMatchers = new Map<
+  string,
+  {
+    isMatch: (str: string) => boolean;
+    negated: boolean;
+  }
+>();
+
+// Every time micromatch is called, it will parse the glob strings and turn them
```

Comment:
One of the windows builds was flaking and there's no way to re-run, so I decided to add some code comments to this file.